tests/dep-time-travel.sh

```diff
@@ -8,19 +8,23 @@ export NPM_CONFIG_LEGACY_PEER_DEPS=true
 
 npm install --no-save "eslint@${ESLINT_VERSION}" --ignore-scripts
 
-# completely remove the new TypeScript parser for ESLint < v5
-if [[ "$ESLINT_VERSION" -lt "5" ]]; then
-  echo "Removing @typescript-eslint/parser..."
-  npm uninstall --no-save @typescript-eslint/parser
-fi
-
-# use these alternate TypeScript dependencies for ESLint < v4
-if [[ "$ESLINT_VERSION" -lt "4" ]]; then
-  echo "Downgrading babel-eslint..."
-  npm i --no-save babel-eslint@8.0.3
-
-  echo "Downgrading TypeScript dependencies..."
-  npm i --no-save typescript-eslint-parser@15 typescript@2.8.1
+re="^[0-9]+$"
+
+if [[ "$ESLINT_VERSION" =~ $re ]] ; then
```

Comment:
i'm not a fan of this approach; i'd rather hardcode the temporary 8 prerelease semver string, which we definitely aren't going to publish a release supporting.

Reply:
I think you're right. But I wanted to check if CI works at this point.